ros2cli/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
```

Comment:
You probably don't want this file included.

Reply:
b8ff16d